Automattic/wp-calypso
# Upload Plugins: Fix Back Button Infinite Loop

## Proposed Changes

If you currently upload a plugin and there's an error (eg. uploading a ZIP file that contains no plugin), the "Back" button creates an infinite loop because it returns you to that error, and the new upload never finishes. You're forced to restart Calypso.

<img width="924" alt="Screenshot 2024-02-13 at 22 54 11" src="https://github.com/Automattic/wp-calypso/assets/43215253/f5783fa2-1238-43b4-b280-7ddf101a89da">

I think it makes more sense to send folks to the Plugins page, since that's the only screen where they can access the "Upload plugins" button anyway. 

## Testing Instructions

Go to My Sites > Plugins, and click "Upload plugin" (top right corner). Confirm the "Back" button works, including in the scenario I described above.

<img width="800" alt="Screenshot 2024-02-13 at 22 55 51" src="https://github.com/Automattic/wp-calypso/assets/43215253/baa88e4d-e326-4a34-93e8-b8847da389c5">

cc @lupus2k, @DustyReagan
